Hakawai/Mentions/HKWMentionsPluginV2.m

```diff
@@ -1006,6 +1007,26 @@ - (BOOL)textView:(__unused UITextView *)textView shouldChangeTextInRange:(NSRang
         [self bleachMentionsIntersectingWithRange:range];
     }
 
+    // If we just did a non-programmatic paste, insert the text directly and add attributes
+    // This is needed because rich text formatting is off, which strips attribution from pasting, which we want to maintain.
+    // Once (if ever) we turn rich text formatting on, we can remove this.
```

Comment:
wasn't the issue only exist for certain pasting flow?  in other cases for pasting, is rich text formatting on